HW10/OlenaBudko/Task_10_2.py

```
@@ -0,0 +1,113 @@
+    def get_density(self):
+        return round(self.mass / self.get_volume(), 5)
+
+
+# 6.
+# Although Timmy had no idea why changing name is so important for his boss, he realized, that it's not the end,
+# so he turned to you, his guru, to help him and asked you to prepare some function, which could change name of given class.
+# Note: Proposed function should allow only names with alphanumeric chars (upper & lower letters plus ciphers),
+# but starting only with upper case letter. In other case it should raise an exception.
+
+def class_name_changer(cls, new_name):
+    setattr(cls, "__name__", new_name)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)